CONTRIBUTING.md

```diff
@@ -57,50 +280,95 @@ add an entry to the NCL Index in cases where a full **NCL Entry** is not
+```
+make clean html
+```
+This will build the documentation locally.
 
-1. [INTEND TO AUTOMATE IN THE FUTURE] Add the new file to the `.rst` file in the
-   same directory in `applications/` as your new file to add it to the webpage's
-   TOC.
+New documentation can be viewed by opening `index.html` generated under
+ `_build/html/index.html` on a local browser
 
-1. [INTEND TO AUTOMATE IN THE FUTURE] Add the new file to
-   the `applications/applications.rst` file to add it to the cards on the main
-   page of the website.
+```
+firefox _build/html/index.html
+```
 
-1. Make sure to clear and run all outputs before asking for a review
+## Contribute the Code
+Once you have completed your changes on notebooks and they are ready for review
+by the GeoCAT team, you can open a new Pull Request. This section describes how
+to open a pull request and what to expect after you open it.
 
-### NCL Entry
+### Check Files Changed
 
-1. Create a new file in `ncl/ncl_entries/` based off of the
-   `templates/ncl_template.ipynb` template.
+For a Python Entry, the files changed should include:
+- New Python page under `applications`
+- Link to directory in `.rst` file in the same directory as `applications/`
+- Link new file in `applications/applications.rst` file
 
-1. Add the file to the `ncl/ncl_entries/ncl_entries.rst` file
+For a NCL to Python entry, the files changes should include:
+- A new receipts entry under `receipts`
+- A NCL to Python notebook under `ncl_entries`
+- A new row in `ncl-index-table.csv`
 
-1. See below for adding the covered functions to the NCL Index
+### Open a new Pull Request
+A Pull Request is a request to merge code from your local fork to the main repository.
+GitHub has extensive [pull request guides and documentation](https://docs.github.com/en/pull-requests)
+if you'd like more information
 
-1. Make sure to clear and run all outputs before asking for a review
+On your local fork of the `geocat-applications` repo:
+1. Open "Contribute" tab
+2. Select "Open Pull Request"
+3. Select "New Pull Request" button
 
-### Receipt
+This will open page to compare your local changes to the main repository. Make sure that
+the `base repository` is `NCAR/geocat-applications` and the `base` branch is set to `main`.
+This will generate a pull request comparing your local changes to the  main branch of the
+main `geocat-applications` repository. The `head repository` and `compare` branch should
+be your forked repo and the branch that contains your changes respectively.
 
-1. Create a new file in `ncl/ncl_receipts/` based off of the
-   `templates/receipt_template.ipynb` template.
+The pull request form will display all the changes you've made on the branch compared to
+the main repo. We recommend adding a short descriptive title to the pull request (e.g.
+"Add Python datetime and days_in_month"). In the body of the pull request please list
+out the NCL or Python functions being covered with a short description of the notebooks.
```

Comment:
Or maybe we remove it here and just add it to the PR template?  I don't know that folks need a warning that this will be requested.  Plus then we just have to maintain it in one spot that is closer to where it's being used.